### PR TITLE
Add source_type to export

### PR DESCRIPTION
**Pull request by kmcelwee**
Addresses #703

### CODE REVIEW

File: mep/accounts/management/commands/export_events.py [hunk 1]
```diff
@@ -172,7 +172,8 @@ def item_info(self, event):
     def source_info(self, footnote):
         '''source details from a footnote'''
         source_info = OrderedDict([
-            ('citation', footnote.bibliography.bibliographic_note)
+            ('citation', footnote.bibliography.bibliographic_note),
+            ('source_type', footnote.bibliography.source_type.name)
```
Comment (rlskoeser) [inline mep/accounts/management/commands/export_events.py:176]: This is an ordered dict so we can control the json output order, and I would like it to match the csv field order — so please put source type before citation.

**Review by rlskoeser — APPROVED** [review]
Great. Just one small order thing that I forgot to tell you about.

**Comment by rlskoeser** [comment]
@kmcelwee should #703 be a 0 instead of a 1? trivial or no?

**Comment by kmcelwee** [comment]
@rlskoeser Your instructions were clear, so it was pretty trivial for me. I agree it should be a 0, unless writing up the dev notes was complicated?

File: mep/accounts/management/commands/export_events.py (hunk 1 above)
Comment (kmcelwee) [reply mep/accounts/management/commands/export_events.py]: Makes sense! fixed in f488806

**Comment by rlskoeser** [comment]
> @rlskoeser Your instructions were clear, so it was pretty trivial for me. I agree it should be a 0, unless writing up the dev notes was complicated?

No, writing the notes was pretty easy. Please change the estimate on the issue and add a comment documenting why.

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**